lckernel/cad/math/intersect.cpp

```diff
@@ -128,14 +118,18 @@ bool Intersect::operator()(const lc::geo::Circle& c, const lc::entity::Arc& arc)
     return false;
 }
 
-bool Intersect::operator()(const lc::geo::Circle& c, const lc::entity::Ellipse& e) {
-    // TODO: test if point is on path
-    auto &&coords = maths::Intersection::QuadQuad(c.equation(), e.equation());
-    if (!coords.empty()) {
-        std::cerr << __PRETTY_FUNCTION__ << " TODO Check if point's are on path" << std::endl;
-    }
-    for (auto &i : coords) {
-        _intersectionPoints.push_back(i);
+bool Intersect::operator()(const lc::geo::Circle& c, const lc::entity::Ellipse& arc) {
```

Comment:
Don't call your `Ellipse` `arc` :)
```suggestion
bool Intersect::operator()(const lc::geo::Circle& c, const lc::entity::Ellipse& ellipse) {
```